.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: node_js
 node_js:
-  - 4
-  - 6
+  - '4'
+  - '6'
+  - '8'
+  - '10'
```

Comment:
I've added node 8 and 10 since they're technically supported.

https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md